client/Assets/Scripts/CustomLevelManager.cs

```diff
@@ -196,6 +201,19 @@ IEnumerator CameraCinematic()
             yield return new WaitForSeconds(2.1f);
             CancelInvoke("Substract");
             InvokeRepeating("MoveYCamera", 0.3f, 0.1f);
+            Utils
+                .GetAllCharacters()
+                .ForEach(character =>
+                {
+                    character.characterBase.ToggleSpawnFeedback(true, character.PlayerID);
```

Comment:
Why not changing the `spawnFeedback` here?
```suggestion
                    character.characterBase.spawnFeedback.SetActive(true);
```

The same goes for the function call below